settings/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	extends: 'plugin:@wordpress/eslint-plugin/recommended',
+
+	rules: {
+		'jsdoc/require-param-type': 0,
+		'prettier/prettier': [
+			'error',
+			{
+				...require( '@wordpress/prettier-config' ),
+				printWidth: 100,
```

Comment:
Default is 80, which resulted in some of the longer strings having `{' '}` inserted. 100 seems like a reasonable middle ground where this is rare.